modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/manager/v1_0/DefaultObjectEntryManagerImpl.java

```diff
@@ -861,15 +862,38 @@ private Map<String, String> _addAction(
 				}
 
 				for (ObjectEntry nestedObjectEntry : nestedObjectEntries) {
+					if (_isFromManySide(
+							objectDefinition, objectRelationship,
+							relatedObjectDefinition)) {
+
+						Map<String, Object> nestedObjectEntryProperties =
+							nestedObjectEntry.getProperties();
+
+						String objectRelationshipName = StringBundler.concat(
+							"r_", objectRelationship.getName(), "_",
```

Comment:
Is iit possible to set this field in the `ObjectRelationshipElementsParser` implementation? I think maybe we need to include a reference to the parent, but that would be enough

Reply:
I'm going to take note of this because it is going to be solved on this https://liferay.atlassian.net/browse/LPS-186732